src/GreaterThanOrEqualsToValidator.js

```diff
@@ -1,19 +1,26 @@
-const {isNumber} = require('lodash');
+const {isDate, isNumber} = require('lodash');
 const Validator = require('./Validator');
+const {parseDate} = require('../src/util/util');
 
 class GreaterThanOrEqualsToValidator extends Validator {
 
     constructor(arg) {
-        if (!isNumber(arg)) {
-            throw new Error('Argument must be a number.');
+        if (!isNumber(arg) && !isDate(arg)) {
+            throw new Error('Argument must be a number or Date.');
```

Comment:
fix error message: `Argument must be a number or a Date instance`

Reply:
Listo.